netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,123 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
-* The *UKNOWN* error code was removed. +
-All unknown errors are now thrown under the *ERROR_FAULT* error code.
-* Many operation inputs were slightly altered, to make them more consistent with other Anypoint Connectors.
 * All exposed classes have been moved from the `org.mule.module.netsuite.extension.api` package to the `com.mulesoft.connector.netsuite.extension.api` package.
 
 [[changed_operations]]
 == Changed Operations
 
+The behavior of some operations has changed in NetSuite Connector 10.0.
+
+The following operations no longer default to `payload`:
+
 [%header%autowidth.spread]
 |===
-|Operation | Parameter | Change
-
-| Change Email a| newEmail a| No longer defaults to `payload`
-| Change Password a| newPasword a| No longer defaults to `payload`
-| Get Data Center URLs a| accountId a| No longer defaults to `payload`
-| Async Search a| content a| Is now a *primary content* parameter
-| Get Async Result a| jobId a| No longer defaults to `payload`
-| Async Add List a| recordAttributes a| Is now a *primary content* parameter
-| Async Update List a| recordAttributes a| Is now a *primary content* parameter
-| Async Upsert List a| recordAttributes a| Is now a *primary content* parameter
-| Update Invitee Status List a| inviteeStatuses a| Is now a *primary content* parameter
-| Get Posting Transaction Summary a| fields a| Is now a *primary content* parameter
-| Get Posting Transaction Summary a| filters a| Is now a *content* parameter
-| Add File a| attributes a| Is now a *content* parameter
-| Add Record a| attributes a| Is now a *content* parameter.
-| Get Custom Record a| internalId a| No longer defaults to `payload` 
-| Get Deleted Records a| deletedDate a| Is now a *content* parameter
-| Update Record a| attributes a| Is now a *content* parameter
-| Upsert Record a| attributes a| Is now a *content* parameter
+|Operation | Parameter
+
+| Change Email a| newEmail
+| Change Password a| newPasword
+| Get Data Center URLs a| accountId
+| Get Async Result a| jobId
+| Get Custom Record a| internalId 
 |===
 
+The following operations are now primary content parameters:
+
+[%header%autowidth.spread]
+|===
+|Operation | Parameter 
+| Async Search a| criteria 
+| Async Add List a| recordAttributes 
+| Async Update List a| recordAttributes 
+| Async Upsert List a| recordAttributes
+| Update Invitee Status List a| inviteeStatuses 
+| Get Posting Transaction Summary | fields 
+| Get Posting Transaction Summary | filters
+| Add File a| attributes 
+| Add Record a| attributes  
+| Get Deleted Records a| deletedDate 
+| Update Record a| attributes 
+| Upsert Record a| attributes 
+|===
+
+When a parameter is designated as primary content, if you do not explicitly assign a value to it, it defaults to `payload`. 
+
+For example, the `criteria` parameter for the Async Search operation is now designated as a primary content parameter. This means that if you do not explicitly assign a value to the `criteria` parameter, it defaults to #[payload].
```

Comment:
How is `payload` different from #[payload]?
If they are both syntax, then they are both in ``.